### PR TITLE
Address some bpf fv flakes

### PR DESCRIPTION
**Pull request by tomastigera**
## Description

    [BPF] ignore map entries that have non-existent values
    
    Fixes this flake in FVs:
    
    Unexpected error:
        <*fmt.wrapError | 0xc00274d380>:
        cannot parse json output: json: cannot unmarshal object into Go struct field mapEntry.value of type []string
        [{
                "key": ["0x01","0x00","0x00","0x00","0x00","0x00","0x00","0x00"
                ],
                "value": {
                    "error": "No such file or directory"
                }
            },{
                "key": ["0x02","0x00","0x00","0x00","0x00","0x00","0x00","0x00"
                ],
                "value": ["0x0a","0x41","0x00","0x02","0x77","0x1f","0x00","0x00"
                ]
            },{
                "key": ["0x00","0x00","0x00","0x00","0x00","0x00","0x00","0x00"
                ],
                "value": ["0xac","0x11","0x00","0x03","0x2b","0x19","0x00","0x00"
                ]
            }
        ]

    Increase fv timeout for felix to become healthy. Takes about 6s

## Todos

- [ ] Tests
- [ ] Documentation
- [ ] Release note

## Release Note

```release-note
TBD
```

## Reminder for the reviewer

Make sure that this PR has the correct labels and milestone set.

Every PR needs one `docs-*` label.

- `docs-pr-required`: This change requires a change to the documentation that has not been completed yet.
- `docs-completed`: This change has all necessary documentation completed.
- `docs-not-required`: This change has no user-facing impact and requires no docs.

Every PR needs one `release-note-*` label.

- `release-note-required`: This PR has user-facing changes. Most PRs should have this label.
- `release-note-not-required`: This PR has no user-facing changes.

Other optional labels:

- `cherry-pick-candidate`: This PR should be cherry-picked to an earlier release. For bug fixes only.
- `needs-operator-pr`: This PR is related to install and requires a corresponding change to the operator.

### CODE REVIEW

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

File: felix/bpf/bpf.go
Comment (fasaxc) [inline felix/bpf/bpf.go:404]: Why suppress this one?

File: felix/bpf/bpf.go
Comment (fasaxc) [inline felix/bpf/bpf.go:394]: I don't see how Err can get set, looks like value would be an object with an error inside it?

Parsing mixed type json is a pain but I might do:
```
var untyped struct {
  Key []string
  Value any
}

err := json.Unmarshal(data, &untyped)
if err != nil {
  ...
}

switch typed := untyped.Value.(type) {
  case map[string]any:
    // An error?
    err = fmt.Errorf("value error: %v", typed["error"])
  case []any:
    // A value
  ...
}
```

File: felix/bpf/bpf.go
Comment (fasaxc) [inline felix/bpf/bpf.go:437]: Why suppress the error?

**Review by fasaxc — CHANGES_REQUESTED** [review]

File: felix/bpf/bpf.go
Comment (tomastigera) [reply felix/bpf/bpf.go:404]: we cannot skip over some entries, so a poor mans solution saying, the entry does not look like we expect so just return an empty entry which we later on filter out. But I guess I will go with proper parsing as you suggest above.

File: felix/bpf/bpf.go
Comment (tomastigera) [reply felix/bpf/bpf.go:394]:
```
  case map[string]any:
    // An error?
    err = fmt.Errorf("value error: %v", typed["error"])
```
I think we should not return an error when we see an error on an entry.

> I don't see how Err can get set

The output might have changed and is not longer settable. Apparently an error is delivered in a map[string]string as "error" now

**Comment by marvin-tigera** [comment]
Removing "merge-when-ready" label due to new commits

**Review by fasaxc — APPROVED** [review]

**Pull request merged**
